statsmodels/tsa/stattools.py

```diff
@@ -1330,10 +1330,10 @@ def grangercausalitytests(x, maxlag, addconst=True, verbose=True):
     >>> import numpy as np
     >>> data = sm.datasets.macrodata.load_pandas()
     >>> data = data.data[["realgdp", "realcons"]].pct_change().dropna()
-
+    >>>
     # All lags up to 4
     >>> gc_res = grangercausalitytests(data, 4)
-
+    >>>
```

Comment:
Same here.  Gaps are find, but there shouldn't be a commend below. So please remove the #.